### DISCUSSION TITLE
Hardware Detection Troubleshooting

### DISCUSSION THREAD

**Discussion by Fatjoeny** [linuxmint/discussions#685] [Development]
Good day, dear Linux Mint development team. Your operating system is still not up to date with the Linux kernel and still does not recognize any different WLAN USB stick adapters (because the internal WLAN adapter is broken) and it also does not recognize many different printer models!!!!!. If you could PLEASE fix these errors, then your operating system would be the No. 1 best alternative in the world to Windows 11 Pro operating system. Thank you very much for your efforts to correct these errors!!!.

Best Regards

**Comment by Fantu** [comment]
In case you need more recent kernel there are backports available on both Ubuntu and Debian (for LMDE). Some wlan adapter may also require external firmware.
About printers there are also drivers and software out of kernel.
Needs to know at least brand and model of hardware not recognized to make possible research what you need.
